<?xml version="1.0" encoding="UTF-8"?>
<launch>
  
    <param name="intrinsic_path"    value="$(find automatic-camera-pointcloud-calibration)/data/parameters/intrinsic.txt" />  <!-- intrinsic file -->
    <param name="input_photo_path"  value="$(find automatic-camera-pointcloud-calibration)/data/photo/3.jpg" />  <!-- photo to find the corner -->
    <param name="output_path"        value="$(find automatic-camera-pointcloud-calibration)/data/feature/corner_photo.txt" />  <!-- file to save the photo corner -->
    <param name="binary_threshold"        value="80" />  <!-- Change it when the feature extraction works not well! -->
    <param name="min_area"        value="40" />  <!-- Depends on the pixel area of squares -->
    <param name="max_area"        value="100" />  <!-- Depends on the pixel area of squares -->
    <param name="rectangularity_preset"        value="0.95" />  <!-- minimal it when your squares are abnormal -->
    <param name="circularity_preset"        value="0.9" />  <!-- It decides whether your features are like circles -->


    <node pkg="automatic-camera-pointcloud-calibration" name="feature_extraction_camera" type="feature_extraction_camera" output="screen"></node>

</launch>

